Sources/Turnstile/Credentials/Credentials.swift

```diff
@@ -13,4 +13,4 @@
  */
 public protocol Credentials { }
 
-public struct Sesssion: Credentials {}
+public struct Session: Credentials {}
```

Comment:
That's actually a great idea. We should test it to make sure it doesn't break anything, but can't think of any reason it should right now. I'll get on this tomorrow afternoon.